ui/apps/platform/src/Containers/Vulnerabilities/WorkloadCves/components/ExceptionRequestModal/ExceptionRequestModal.tsx

```diff
@@ -14,21 +21,58 @@ export type ExceptionRequestModalProps = {
     type: CveExceptionRequestType;
     cves: DeferralFormProps['cves'];
     scopeContext: ScopeContext;
+    onExceptionRequestSuccess: (vulnerabilityException: VulnerabilityException) => void;
     onClose: () => void;
 };
 
-function ExceptionRequestModal({ type, cves, scopeContext, onClose }: ExceptionRequestModalProps) {
+function ExceptionRequestModal({
+    type,
+    cves,
+    scopeContext,
+    onExceptionRequestSuccess,
+    onClose,
+}: ExceptionRequestModalProps) {
     const cveCountText = pluralize(cves.length, 'workload CVE');
     const title =
         type === 'DEFERRAL'
             ? `Request deferral for ${cveCountText}`
             : `Mark ${cveCountText} as false positive`;
 
+    const { mutate, error: deferralError } = useRestMutation(createDeferralVulnerabilityException);
+
+    function onDeferralSubmit(formValues: DeferralValues, helpers: FormikHelpers<DeferralValues>) {
+        if (formValues.expiry) {
```

Comment:
In order to avoid setting a default `expiry` or having to deal with `null` values down the call stack, we validate on form submission so that the error can be delegated to formik.